src/scss/constants.scss

```diff
@@ -108,7 +108,7 @@ $z-layers: (
 
 @function z($layer) {
   @if not map-has-key($z-layers, $layer) {
-    @warn "No layer found for `#{$layer}` in $z-layers map. Property omitted. Falling back to 1.";
+    @error "No layer found for `#{$layer}` in $z-layers map.";
     @return 1;
```

Comment:
```suggestion
    @error "No layer found for `#{$layer}` in $z-layers map.";
```

Reply:
With this `@error` change, if you try to use a `z(doesnt-exist)` or similar, how does the error show? Hopefully in an obvious way.

Reply:
Just checked this, looks good, and blocks the app from starting, so nice and obvious!

![image](https://user-images.githubusercontent.com/753891/109119134-5df4f100-7798-11eb-812d-36b719adc927.png)

---

Removed the extra `@return` in https://github.com/sparkletown/sparkle/pull/1202/commits/f19c6c45d102faaface23b02282aec6b2db32c5e